CPRynner/CPRynner.py

```diff
@@ -3,130 +3,114 @@
 clusterview and runnoncluster plugins.
 """
 
+from future import *
```

Comment:
Two issues here:

1. `import *` imports are to be avoided; the `python-future` manual has a guide on how to import more cleanly.
2. What is `future` being used for? Are we still targeting Python 2?